critiquebrainz/frontend/views/avg_rating.py

```diff
@@ -0,0 +1,12 @@
+import critiquebrainz.db.avg_rating as db_avg_rating
+import critiquebrainz.db.exceptions as db_exceptions
+
+def get(entity_id, entity_type):
```

Comment:
It would probably be better to just put this into `__init__.py` of the `frontend.views` package instead of creating a new module just for one utility function. I think it's expected that modules in `views` contain actual views.

Reply:
Move the function to __init__.py